src/main/flight/mixer.c

```diff
@@ -789,12 +791,16 @@ FAST_CODE_NOINLINE void mixTable(timeUs_t currentTimeUs, uint8_t vbatPidCompensa
 
     // Find roll/pitch/yaw desired output
     float motorMix[MAX_SUPPORTED_MOTORS];
-    float motorMixMax = 0, motorMixMin = 0;
+    float motorMixMax = 0, motorMixMin = 0; 
+    extern float dT;
+    integratedYaw =
```

Comment:
Isn't this just easier to apply earlier:
`integratedYaw = constrainf(integratedYaw + pidData[FD_YAW].Sum , -currentPidProfile->pidSumLimitYaw, currentPidProfile->pidSumLimitYaw);`